src/test/java/org/jabref/logic/l10n/LocalizationParser.java

```diff
@@ -75,6 +75,14 @@
         }
     }
 
+    public static Set<LocalizationEntry> findLocalizationParametersStringsInJavaFiles(LocalizationBundleForTest type)
+            throws IOException {
+        return Files.walk(Paths.get("src/main"))
```

Comment:
Just looked at it. Doesn't seem like a real change to me except that its a parameter then?!